package/gluon-client-bridge/luasrc/lib/gluon/upgrade/320-gluon-client-bridge-wireless

```diff
@@ -13,13 +14,19 @@ local function is_disabled(config, name)
 	return config.disabled(false)
 end
 
+local has_client_radio = false
```

Comment:
should be renamed to `has_client_only_radio` as this checks for a radio which can not be used for mesh